pandas/_typing.py

```diff
@@ -22,3 +23,5 @@
                              Timedelta)
 Dtype = Union[str, np.dtype, ExtensionDtype]
 FilePathOrBuffer = Union[str, Path, IO[AnyStr]]
+Scalar = Union[int, float, str]
+DatetimeScalar = Union[Scalar, datetime]
```

Comment:
I wouldn't do this here as its too general

Reply:
ok